crates/burn-tensor/src/tensor/distribution.rs

```diff
@@ -107,3 +108,16 @@ impl Distribution {
         DistributionSampler::new(kind, rng)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_distribution_default() {
+        let dist: Distribution = Default::default();
+
+        assert_eq!(dist, Distribution::Default);
+        assert_eq!(Distribution::default(), Distribution::Default);
+    }
+}
```

Comment:
Probably this test is not needed.

Reply:
I see the point of unittests not as verifying behavior; but as hardening a codebase against automated refactoring breakage. There should be something in the tests that exercise those 3 interfaces; it's easy to add, and it potentially prevents breakages under rewrites later.